docs/en/Troubleshoopting.md

```diff
@@ -73,4 +73,4 @@ SendResult [sendStatus=SEND_OK, msgId=0A42333A0DC818B4AAC246C290FD0000, offsetMs
 ```
 
 - msgId，for the client, the msgId is generated by the producer instance. Specifically, the method `MessageClientIDSetter.createUniqIDBuffer()` is called to generate a unique Id.
-- offsetMsgId, offsetMsgId is generated by the Broker server when writing a message ( string concating "IP address + port" and "CommitLog's physical offset address"), and offsetMsgId is the messageId used to query in the RocketMQ console.
+- offsetMsgId, offsetMsgId is generated by the Broker server when writing a message ( string concat with "IP address + port" and "CommitLog's physical offset address"), and offsetMsgId is the messageId used to query in the RocketMQ console.
```

Comment:
IMO, better use  ` string consisting of "IP address + port" and "CommitLog's physical offset address"`